docs/developer/soffits/README.md

```diff
@@ -0,0 +1,46 @@
+# uPortal Soffits
+
+Soffit is a technology for creating content that runs in [Apereo uPortal][].  It
+is intended as an alternative to [JSR-286 portlet development][].
+
+## Why Would I Want Soffits?
+
+You are a Java web application developer.  You are tasked with developing content
+for [Apereo uPortal][].
+
+You are not excited about doing Java Portlet development
+[in the traditional way][] or even using [Spring Portlet MVC][].  You correctly
+conclude that the Java Portlet APIs are _large_, _obtuse_, and _actively
+interfere_ with contemporary web development practices and frameworks that you
+want to be using.
+
+Apereo Soffit is an alternative approach to producing content for uPortal that
+is not based on JSR-286 or the portlet container.
```

Comment:
Those 2 concepts pre-date JSR-286 and (IMHO) are reasonably applicable to "dashboardy" web applications broadly.  The point is that if -- as a producer of complex content for uPortal -- you want to leave portlets behind, you can do that with Soffits.

You won't need any...

* Portlet API Java classes
* Portlet JSP tag library
* portlet.xml file
* `$ ant deployPortletApp`
* Action/Event/Render/Resource lifecycle

The Soffit approach give you a 100% break -- as a developer of content -- from portlety things.